keyboards/acheron/apollo/87h/gamma/info.json

```diff
@@ -1,122 +1,226 @@
 {
```

Comment:
Same here.

Reply:
Done in [c372580](https://github.com/qmk/qmk_firmware/pull/20539/commits/c372580a521b857fafe6c2a83eea4eddeecba55e).